seed_db/seed/Proposal.json

```diff
@@ -9,8 +9,8 @@
 		"lastname": "Schlepütz",
 		"title": "Full-volume high-resolution x-ray tomographic microscopy reconstruction of a juvenile rat lung in fresh post mortem conditions; cropped to the maximum extent of the lung structure",
 		"abstract": "",
-		"ownerGroup": "aGroup",
-		"accessGroups": [],
+		"ownerGroup": "group1",
+		"accessGroups": ["group2"],
```

Comment:
if you want to change the groups here, rather than in the .env, should group2 belong to `ACCESS_GROUPS_STATIC_VALUES`? or it's enough that it is in the `CREATE_DATASET_GROUPS`?